### ISSUE TITLE
Tacotron: Trying r < 5

### ISSUE THREAD

**Issue by erogol** [mozilla/TTS#17]
Expecting better fidelity with r=2, which is also the setting used by the original paper. 

Our previous runs use r=5 for the benefit of faster training.

**Comment by erogol** [comment]
with r=2 generated speech is much better for the training set but it lacks fidelity at the validation set. And the loss curse shows strong overfitting.

**Comment by erogol** [comment]
r=3 is running...
